app/pages/project/instances/instance/tabs/NetworkingTab.tsx

```diff
@@ -337,9 +339,16 @@ export function NetworkingTab() {
     getCoreRowModel: getCoreRowModel(),
   })
 
-  const disabledReason =
-    eips.items.length >= 32
-      ? 'IP address limit of 32 reached for this instance'
+  const ephemeralDisabledReason =
+    eips.items.filter((ip) => ip.kind === 'ephemeral').length >= 1
+      ? 'Ephemeral IP already attached'
+      : rows.length === 0
```

Comment:
rename `rows` to `nics`. I'm guessing the name is a holdover from when this page only had nics